File: aasexplorer-core/src/main/java/com/ficture7/aasexplorer/client/ParseException.java
package com.ficture7.aasexplorer.client;

/**
 * Exception thrown when there was an issue parsing a document.
 *
 * @author FICTURE7
 */
public class ParseException extends Exception {

    /**
     * Constructs a new instance of the {@link ParseException} class.
     *
     * @param cause Cause.
     */
    ParseException(Throwable cause) {
        super(cause);
    }
}

File: aasexplorer-core/src/main/java/com/ficture7/aasexplorer/client/ParseException.kt
package com.ficture7.aasexplorer.client

/**
 * Exception thrown when there was an issue parsing a document.
 *
 * @author FICTURE7
 */
class ParseException : Exception {

    /**
     * Constructs a new instance of the [ParseException] class.
     *
     * @param cause Cause.
     */
    internal constructor(cause: Throwable) : super(cause)
}
